tests/test_init.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+. ../scripts/init
+
+echo "*** test file: $(basename ${BASH_SOURCE[0]})"
+
+test_use_cmd() {
+  devenv add foo
+  devenv use foo
+  assertNotNull ${DEVENVFLAVOR}
```

Comment:
I think using `assertNotContains` to check the result would be better

Reply:
What should be the value of `$DEVENVFLAVOR` after `devenv use foo`?  If it should be `foo`, perhaps this may use `assertEqual` or something like it?

Reply:
I have refine the test case in https://github.com/solvcon/devenv/pull/162/commits/9ca221e21533b7f6f2b109f4596fee1f814e1c73

Reply:
It's great.  Thanks.